### ISSUE TITLE
Enable classification metrics to use logits and class indicators

### ISSUE THREAD

**Issue by afoncubierta** [BiomedSciAI/fuse-med-ml#215]
The current implementation of the classification metrics expects class labels as integers. The user might want to use a model that outputs logits to estimate the class. 

In this case, it would be possible to spare the user from tuning the model, by enabling the metric computation to use logits or class scores directly. This can be easily done by using type_of_target function from sklearn and if the result is some sort of `continous` value, apply an argmax function.

An alternative is that the user will implement the argmax or even explore the operating point of the model, but the proposed solution would be the default in case the user takes no action.
